components/Typography/src/MDCTypography.m

```diff
@@ -178,6 +178,13 @@ - (UIFont *)italicFontOfSize:(CGFloat)fontSize {
   return [UIFont italicSystemFontOfSize:fontSize];
 }
 
+- (UIFont *)boldItalicFontOfSize:(CGFloat)fontSize {
+  UIFont *bold = [self regularFontOfSize:fontSize];
```

Comment:
This naming seems a bit surprising.  bold = regularFont?  And there is no type identifier.
Consider s/bold/regularFont/